mmcv/runner/hooks/checkpoint.py

```diff
@@ -28,6 +33,18 @@ class CheckpointHook(Hook):
             regardless of interval.
         sync_buffer (bool): Whether to synchronize buffers in different
             gpus. Default: False.
+        file_client_args (dict): Arguments to instantiate a FileClient.
+            See :class:`mmcv.fileio.FileClient` for details. Default: None.
+            `New in version 1.3.15.`
+
+    .. warning::
+        Before v1.3.15, the `out_dir` argument indicates the path where the
+        checkpoint is stored. However, in v1.3.15 and later, `out_dir`
```

Comment:
Change
```
Before ..... However ....... and the last level directory of `runner.work_dir`.
```
to
```
Before v1.3.15, ``out_dir`` indicates the file path the checkpoint to be saved.
Since v1.3.15, ``out_dir`` indicates the root directory rather than the checkpoint path. The final path to save the checkpoint is inferred by xxx. (You may suggest why we do this).
```